linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_immutable/rule.yml

```diff
@@ -56,20 +56,22 @@ references:
     stigid@ol8: OL08-00-030121
     stigid@rhel8: RHEL-08-030121
 
+ocil_clause: 'the audit system is not set to be immutable by adding the "-e 2" option to the end of "/etc/audit/audit.rules"'
+
 ocil: |-
     Verify the audit system prevents unauthorized changes with the following command:
-
+    <pre>
     $ sudo grep "^\s*[^#]" /etc/audit/audit.rules | tail -1
-
+    </pre>
+    <pre>
```

Comment:
This cause too many vertical whitespace around the `-e 2` line. You probably should remove the newlines after opening pre tags and before closing pre tags.